README.md

```diff
@@ -364,6 +366,16 @@ Dodge et. al 2015. Tests Memory Networks on 4 tasks including reddit dialog task
 * [This TextSum model](https://github.com/tensorflow/models/tree/master/research/textsum) is from the official TensorFlow repository.
 * [TextTeaser](https://github.com/DataTeaser/textteaser) is a automatic text summarization algorithm. The algorithm is ported in Python and Scala.  
 
+### Text Classification
+
+[Back to Top](#contents)
+
+* [Convolutional Neural Networks for sentence classfication](https://arxiv.org/pdf/1408.5882v2.pdf) by Kim Yoon is now regarded as the standard baseline for text classification architecture. 
+* [Using a CNN for text classification in TensorFlow](http://www.wildml.com/2015/12/implementing-a-cnn-for-text-classification-in-tensorflow/) by Denny Britz uses the same dataset as Kim Yoon's paper(mentioned above). The code implementation can be found [here](https://github.com/dennybritz/cnn-text-classification-tf).
+* [Facebook's fasttext](https://github.com/facebookresearch/fastText) is a library for fast text representation and text classification
```

Comment:
Maybe you wanted to say something along the lines of: `fastText is a library for text embeddings and text classification`? which is more descriptive and helpful than `*fastText* is a library for *fast text* representation` ?

Reply:
agreed

Reply:
Yes, that what I meant :smile: